dandi/tests/fixtures.py

```diff
@@ -488,6 +488,27 @@ def docker_compose_setup() -> Iterator[dict[str, str]]:
                 raise RuntimeError("Django container did not start up in time")
         yield {"django_api_key": django_api_key}
     finally:
+        if auditfile := os.environ.get("DANDI_TESTS_AUDIT_CSV", ""):
+            with open(auditfile, "wb") as fp:
+                run(
+                    [
```

Comment:
codecov still reports this block not covered by tests -- any idea why?

Reply:
I think that's just due to the annotations on GitHub not being updated.  If I go to [the Codecov site](https://app.codecov.io/github/dandi/dandi-cli/commit/9abc18ff93caa0e6d006ab3137ee1256c1eef40e/blob/dandi/tests/fixtures.py), the lines are marked covered.